extension/rikaicontent.ts

```diff
@@ -65,7 +65,6 @@ type Rikaichan = {
 };
 
 class RcxContent {
-  private dictCount = 3;
```

Comment:
After adding "noUnusedLocals" to the tsconfig.json file, it told me that this dictCount property was not used, so I removed it. I double-checked to make sure that this dictCount property wasn't being used anywhere by searching the code (https://github.com/search?q=repo%3Amelink14%2Frikaikun%20dictCount&type=code).